lib/generators/mountain_view/extra_pages_generator.rb

```diff
@@ -0,0 +1,39 @@
+require_relative './mountain_view_generator_base'
+
+module MountainView
+  module Generators
+    class ExtraPagesGenerator < MountainViewGeneratorBase
+      desc "Generates extra pages and controller, routes"
+
+      def create_extra_pages_views
+        MountainView.configuration.extra_pages.each do |page|
+          file_name = "app/views/mountain_view/styleguide/#{page}.html.#{template_engine}"
+          next if File.exist?(file_name)
+          create_file file_name
+        end
+      end
+
+      def create_mountain_view_controller
+        file_name = "app/controllers/mountain_view/styleguide_controller.rb"
+        create_file  file_name do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.